File: problemSet6/sentimental-mario-more/mario.py
from cs50 import get_int

while True:
    n = get_int("Height: ")
    if 0 < n < 9:
        break

for i in range(n):
    for j in range(n + i + 3):
        if i + j < n - 1 or j == n or j == n + 1:
            print(" ", end="")
        else:
            print("#", end="")
    print()

# FOR INVERTED MARIOMORE
# for i in range(n-1, -1, -1):

File: week1/mario.c
#include <cs50.h>
#include <stdio.h>

int main(void)
{
    int n;
    do
    {
        n = get_int("Size: ");
    }
    while (n < 1);
    for (int i = 0; i < n; i++)
    {
        for (int j = 0; j < n; j++)
        {
            printf("#");
        }
        printf("\n");
    }
}


// int main(void)
// {
//     for (int i = 0; i < 4; i++)
//     {
//         for (int j = 0; j < 8; j++)
//         {
//             printf("#");
//         }
//         printf("\n");
//     }
// }